extension/hook.js

```diff
@@ -1,25 +1,29 @@
 const getManifest = chrome.runtime.getManifest;
-const version = (getManifest && getManifest().version) || 'electron-version';
+const version = (getManifest && getManifest().version) || "electron-version";
```

Comment:
Make sure to use single quotes everywhere - update this in your editor if you can.